internal/payloads/reservation_payload.go

```diff
@@ -42,6 +42,10 @@ type GenericReservationResponsePayload struct {
 	Success *bool `json:"success" nullable:"true"`
 }
 
+type InstanceResponse struct {
+	*models.ReservationInstance
+}
+
```

Comment:
This will cause conflict with @amirfefer 's PR but this will likely go in earlier because there is no IQE test breakage here. Sorry :-)